IFC4x3/Properties/f/FaultPitchAngle_2amUo6QL91DeGgZLQHgMyf/Documentation.md

```diff
@@ -0,0 +1 @@
+Angle between the strike of the slip surface and the slip vector (striation).
```

Comment:
Hard to understand what this will be in 3D.

Reply:
I added as a (long) NOTE:
_"> NOTE&nbsp; In geology, "pitch" and "rake" are terms used to describe the orientation and inclination of geological features, such as folds and faults, with respect to a horizontal plane. Pitch refers to the angle of inclination of a geological feature from the horizontal plane, measured perpendicular to the strike of the feature. It is commonly expressed in degrees or as a ratio of vertical to horizontal distance. For example, if a geological feature has a pitch of 45 degrees, it means that it is inclined at an angle of 45 degrees from the horizontal. Rake, on the other hand, refers to the angle of inclination of a geological feature measured parallel to the strike of the feature. It is also commonly expressed in degrees or as a ratio of vertical to horizontal distance. Rake is often used to describe the orientation of faults, where it represents the angle between the fault plane and a horizontal plane. Both pitch and rake are important parameters in geology as they can provide information about the deformation history and the mechanical behavior of rocks in response to tectonic forces."_